audit-connector/src/main/java/com/sequenceiq/cloudbreak/audit/AuditClient.java

```diff
@@ -151,4 +154,19 @@ private AuditBlockingStub newStub(ManagedChannel channel, String requestId, Stri
         return AuditGrpc.newBlockingStub(channel)
                 .withInterceptors(new AltusMetadataInterceptor(requestId, actorCrn));
     }
+
+    public void listEvents(ListAuditEvent listAuditEvent) {
```

Comment:
might worth getting the latest audit.proto from thunderhead repo and update client here
also, integrate the latest audit service docker image build int `cbd`.

Reply:
Can you create a jira ticket for the upgrade?